Community Balance Patch/Modular Elements/Corporations/Corporations.xml

```diff
@@ -823,10 +823,6 @@
 			<YieldChange>2</YieldChange>
 		</Row>
 	</Policy_BuildingClassYieldChanges>
-	INSERT="" INTO Policy_BuildingClassYieldChanges
```

Comment:
so I found this in the middle of the file, was this file broken this entire time? In that case, is it even needed?

Reply:
Possibly? Could be a recent merge issue.

Reply:
I'm no github expert, but I think it says those changes were made 2 years ago? The last time it was updated was 2 months ago so at the very least it's been like that for 2 months.
Why are there two different places for the corporation code anyway? There's `Community-Patch-DLL\Community Balance Patch\Modular Elements\Corporations` and also `Community-Patch-DLL\Corporations\Corporations`

Reply:
second is a standalone project that was aborted by someone else

Reply:
Should we still have the files then? It hasn't been updated for a while anyway, and if needed it can be accessed but for now I think we can delete it.